examples/Bremsstrahlung/README.rst

```diff
@@ -0,0 +1,48 @@
+Emission of Bremsstrahlung from Laser-foil interaction
+======================================================
+
+
+* author:      Heiko Burau <h.burau (at) hzdr.de>
+* maintainer:  Heiko Burau <h.burau (at) hzdr.de>
+
+This is a simulation of a flat solid density target hit head-on 
```

Comment:
In those files, keep latex style:
- no end of line: soft-wrapping is fine in editors
- one sentence per line